.eslintrc.js

```diff
@@ -0,0 +1,65 @@
+module.exports = {
+    root: true,
+    parser: '@typescript-eslint/parser',
+    plugins: [ '@typescript-eslint' ],
+    extends: [
+        'eslint:recommended',
+    ],
+    env: {
+        node: true,
+        es6: true,
+    },
+    globals: {
+        describe: true,
+        xdescribe: true,
+        fdescribe: true,
+        it: true,
+        xit: true,
+        fit: true,
+        expect: true,
+        $: true,
+        $$: true,
+        driver: true,
+        beforeEach: true,
+        browser: true,
```

Comment:
How about adding the wdio eslint plugin: https://www.npmjs.com/package/eslint-plugin-wdio .. then we could remove these too

Reply:
Done